rest/database/db.go

```diff
@@ -29,6 +28,17 @@ func Init() {
 
 	DB, err = gorm.Open(dialector, &gorm.Config{})
 
+	// Migration/Creation of data tables for DB
+	if !DB.Migrator().HasTable(&models.UserIdentity{}) {
+		DB.Migrator().CreateTable(&models.UserIdentity{})
+	}
+	if !DB.Migrator().HasTable(&models.FavoritePage{}) {
+		DB.Migrator().CreateTable(&models.FavoritePage{})
+	}
+	if !DB.Migrator().HasTable(&models.LastVisitedPage{}) {
```

Comment:
This model does not exist yet.

Reply:
Yeah this file change went past me when I stashed the other changes, I'm changing it back rn